app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  def return_last_three_posts
+    posts.last(3)
+  end
```

Comment:
- [ ] <b>REQUIRED:</b> Kindly use the command `order` to make sure that you are getting the last 3 posts by date.

```ruby
posts.order('created_at Desc').limit(3)

Reply:
This change has been done correctly